.env

```diff
@@ -75,7 +75,7 @@ _APP_MAINTENANCE_INTERVAL=86400
 _APP_MAINTENANCE_RETENTION_EXECUTION=1209600
 _APP_MAINTENANCE_RETENTION_ABUSE=86400
 _APP_MAINTENANCE_RETENTION_AUDIT=1209600
-_APP_USAGE_AGGREGATION_INTERVAL=30
+_APP_USAGE_AGGREGATION_INTERVAL=15
```

Comment:
On purpose or left-over? If on purpose, should we update it in production .env too?

Reply:
on purpose to make the waiting time on tests lesser, not in production only for tests.